backend/app/services/implementations/residents_service.py

```diff
@@ -82,13 +82,29 @@ def delete_resident(self, resident_id):
             )
         db.session.commit()
 
-    def get_residents(self, resident_id=None):
+    def get_residents(self, return_all, page_number, results_per_page=10, resident_id=None):
```

Comment:
Nit: Don't need this defaulted to 10 if it's already being defaulted to 10 in the route. Can probably leave this though

Reply:
Good point ; removed